src/sales-channels/amazon/pricing-products.md

```diff
@@ -10,6 +10,9 @@ There are two types of pricing rules:
 - [Standard Pricing Rule]({% link sales-channels/amazon/standard-price-rules.md %})
 - [Intelligent Repricing Rule]({% link sales-channels/amazon/intelligent-repricing-rules.md %})
 
+   {:.bs-callout .bs-callout-warning}
+   Intelligent repricing rules do not function properly if the Amazon region is set to `Inactive` status, as it is during onboarding. Your pricing calculations depend on your shipping rates, and your region must be in `Active` status for your shipping rates to sync from Amazon. <br/><br/>To update your region status in your Amazon account, go to Settings > Account Info > Vacation Settings. Refer to [Amazon: Listing Status for Vacations](https://sellercentral.amazon.com/gp/help/help.html?itemID=200135620&amp;language=en_MX&amp;ref=ag_200135620_cont_191){:target="_blank"}.
```

Comment:
You don't need `{:target="_blank"}` for links that go outside of the magento primary domain.

Reply:
thank you.  will correct